vocabulary-draft/source-code-version-control.md

```diff
@@ -0,0 +1,36 @@
+# Source Code Version Control Events
+
+This phase includes events related to changes in Source Code repositories that are relevant from a Continuous Delivery perspective.
+
+
+## Repository Events
+
+These events are related to Source Code repositories
+- **Repository Created Event**: a new Source Code Repository was created to host source code for a project
+- **Repository Modified Event**: a Source Code Repository modified some of its attributes, like location, or owner
+- **Repository Deleted Event**: a Source Code Repository was deleted and it is not longer available
+- **Branch Created Event**: a Branch inside the Repository was created 
```

Comment:
Should we have some event for announcing that a branch has been modified, i.e. it now points to a different commit?

Reply:
@erkist that is a good example of something that I think is extremely useful but it might not be necessary to include the initial set of events. How do we document these things to not loose track of them, but at the same time don't push for all these events to be present in the first iteration of the vocabulary (to reduce mental overhead)